From 6e2a723ef54ee2e739c34786981b2c508db803c1 Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Wed, 29 Oct 2008 03:24:18 +0000
Subject: [PATCH] config-allow-gta02-usb-hiddev.patch

Tick requested that we allow USB HIDDEV config on GTA02
so he can strap one on some robot legs he is working on.
He found that having it as a module broke gadget support
somehow (?) so HID and HIDDEV are changed to in-kernel

Signed-off-by: Andy Green <andy@openmoko.com>
---
 arch/arm/configs/gta02-moredrivers-defconfig |    2 +-
 defconfig-gta02                              |    4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/gta02-moredrivers-defconfig b/arch/arm/configs/gta02-moredrivers-defconfig
index c337ecf..add419a 100644
--- a/arch/arm/configs/gta02-moredrivers-defconfig
+++ b/arch/arm/configs/gta02-moredrivers-defconfig
@@ -1291,7 +1291,7 @@ CONFIG_HID=y
 CONFIG_USB_HID=y
 # CONFIG_USB_HIDINPUT_POWERBOOK is not set
 # CONFIG_HID_FF is not set
-# CONFIG_USB_HIDDEV is not set
+CONFIG_USB_HIDDEV=y
 CONFIG_USB_SUPPORT=y
 CONFIG_USB_ARCH_HAS_HCD=y
 CONFIG_USB_ARCH_HAS_OHCI=y
diff --git a/defconfig-gta02 b/defconfig-gta02
index 91d3140..70bdc3e 100644
--- a/defconfig-gta02
+++ b/defconfig-gta02
@@ -1195,10 +1195,10 @@ CONFIG_HID=y
 #
 # USB Input Devices
 #
-CONFIG_USB_HID=m
+CONFIG_USB_HID=y
 # CONFIG_USB_HIDINPUT_POWERBOOK is not set
 # CONFIG_HID_FF is not set
-# CONFIG_USB_HIDDEV is not set
+CONFIG_USB_HIDDEV=y
 
 #
 # USB HID Boot Protocol drivers
-- 
1.5.6.5

